clients/drcachesim/CMakeLists.txt

```diff
@@ -126,6 +126,14 @@ add_exported_library(directory_iterator STATIC common/directory_iterator.cpp)
 add_dependencies(directory_iterator api_headers)
 target_link_libraries(directory_iterator drfrontendlib)
 
+# Right now drpt2trace only works on Linux x86_64.
+if (LINUX AND X64)
+  # Building drpt2trace in dr requires initializing the libipt submodule first.
+  if (HAVE_LIBIPT)
+    add_subdirectory(drpt2trace)
+  endif (HAVE_LIBIPT)
+endif (LINUX AND X64)
```

Comment:
Might be worth a STATUS message on both sides of the if to let the user know whether drpt2trace is being built.

Reply:
Done.